### DISCUSSION TITLE
Old man braucht Hilfe

### DISCUSSION THREAD

**Discussion by Weiberhaus** [tbnobody/OpenDTU#1831] [Q&A]
Moin zusammen, 
habe in einem leichten Anfall von Wahnsinn, im letzten Herbst eine Edelstahlunterkonstruktion zusammengebraten
und 4 Module (1620Wpeak)in 3,5m Höhe an der Südseite meines Hauses angebracht. (musste meiner Frau versprechen, derartige akrobatische Höchstleitungen zukünftig zu unterlassen.) Zwei Module Module befeuern eine EPP-Solarbox, die zwei Redodo 24V /100AH
tagsüber lädt und nachts über einen HM1500 (an dem die restlichen2 Module hängen) ins Hausnetz einspeist. Gedrosselt über eine openDTU, natürlich auf die 600Watt. Nichts desto trotz, muss ich abends den WR über die DTU eindrosseln, weil sonst der Wechselrichter die 600 Watt (ca. 350W davon an den Energieversorger) zieht,bzw. ins Netz speist. Wie jeder Mensch, verfügt man auch im Alter über einen gewissen Fatalismus, soll heißen: das muss auch einfacher gehen. Also Code vom Energieversorger besorgt, Tasmoto IR ins WLan
eingebunden. Keine Probleme. Smart Micro Solar besorgt, ins WLan eingebunden.Keine Probleme. Bei der Initialisierung der SMS ist mir
erst klar geworden, das ich keine IP-Adresse von der open DTU habe. Ich habe ALLES versucht, was mir Laie und Tante Google so einfällt,
kein Erfolg. DTU neu geflasht, alte Fritzbox wieder installiert, anstelle der jetzigen( 7530 AF);  alle Wlan Geräte abgeschaltet; neues Passwort, usw. ; Abstände eingehalten....Okay habe ich mir gesagt, Hardwarefehler...also neue openDTU. Erfolg:Null
In der Bedienungsanleitung von Smart Micro Solar steht: Da OpenDTU die API Schnittstelle geändert hat, ist ein Smart Micro Solar Betrieb nur bis zur OpenDTU Firmware V24.1.26 möglich.
Kann mir jemand auf die Sprünge helfen, oder muss ich alter Mann jeden abend mit dem Handy den Hohmiles eindrosseln.
Dank für Eure Unterstützung. Da ich meine eigenen Erfahrungen in China gemacht habe, finde ich dieses Projekt wegweisend.

**Comment by Marc9485** [comment]
Gude, in der fritz box unter wlan kann man alle Geräte sehen die bit dieser verbunden sind auch die dtu und da steht auch die ip adresse. Aber dein inet brower müsste die ja auch im adressfeld anzeigen wenn du openDTU aufrufst

**Reply by Weiberhaus** [reply]
Hi Marc, erstmal Dank für Reaktion.
DTU wird in der Fritzbox unter Wlan im Funknetz NICHT angezeigt.....in der Rubrik Funkkanal unter "andere Funknetze in der Umgebung"
wird die DTU angezeigt, jedoch nur mit MAC Adresse. Wenn ich mit dem Handy die DTU im Wlan anspreche erreiche ich die nur unter
192.178.4.1....für mich als Laien heißt das: DTU ist nicht ins Heimnetz eingebunden. Liege ich richtig?

**Reply by Marc9485** [reply]
![image](https://github.com/tbnobody/OpenDTU/assets/134625027/b74ca4a0-ee28-4e50-8314-03c02ee676b5)

**Reply by Marc9485** [reply]
Da wird meine dann angezeigt. In deinem Fall würde ich sagen ja, du hast recht. Ist die vielleicht noch im ap-modus? Mal dtu stromlos machen und wieder einstecken. Dann wlan suchen von der dtu und neu verbinden. 
Wäre jetzt mein Vorschlag

**Reply by Weiberhaus** [reply]
Hi, die Hoffnung stirbt zum Schluss, deinen Vorschlag zum gefühlt 100 - dertsten Mal umgesetzt.....leider Null Erfolg.

**Reply by OpenShorty** [reply]
Poste doch mal die Netzwerkeinstellungen deiner DTU inkl. dem letzten Punkt WLAN Konfiguration (Admin AccessPoint). wie in meinem Beitrag. Es muss ja einen Grund haben warum dein AP nicht geschlossen wird und du nicht ins Wlan kommst.

**Comment by OpenShorty** [comment]
Moin, so wie sich das anhört erreichst du die OpenDTU tatsächlich nur über den eigenen Access Point. Das ist die 192.178.4.1.

Komisch ist nur das der Access Point sich eigentlich iwann abschaltet.

Schaue doch mal bitte was in der OpenDTU unter Einstellungen>Netzwerk>Wlan Konfiguration eingetragen ist. 

Unter Wlan SSID muss der Name deines WLANs eingetragen sein bzw. werden. Kann sein das du es manuell eintragen musst.
WLAN Passwort ist dann ja klar, dein Passwort mit dem du dich mit dein WLAN verbinden kannst
Hostname ist dann der Name der DTU, mit dem sie sich in deinem WLan meldet
DHCP würde ich aktivieren damit deine FRITZ!Box der DTU erstmal eine IP zuweis.

Wichtig ist das in den FRITZ!box Einstellung die Einstellung „neue WLan Geräte zulassen“ aktiviert ist bzw das es zugelassen ist.
Außerdem würde ich dann, wenn die OpenDTU verbunden ist, in den Einstellungen für den WLAN Teilnehmer unter IP Einstellung das Häkchen setzen „immer die gleiche IP zuweisen“. So kannst du dir eine Verknüpfung mit der IP erstellen und brauchst dir keine Gedanken über statische IP’a machen.

Schau mal so ist es bei mir eingestellt 

![IMG_6930](https://github.com/tbnobody/OpenDTU/assets/135330209/9af81926-f171-441b-8a28-d22b1550ebb2)

**Reply by Weiberhaus** [reply]
Moin Jungs, 
mir fällt es gerade wie Käse von den Füßen. Ich hatte/habe ein Riesenbrett vorm Kopf. Bei Marc ist mir der "komische " Name der DTU (5DF6A0)aufgefallen und bei Shorty der ausgegraute Bereich beim Hostnamen. Ich Lui habe es als einen Hinweis auf einen Automatismus angesehen........und NICHTS verändert. Ich muss, wenn ich jetzt richtig liege , also mit der Lupe die Zahl vom Chip erkunden und die letzten 6 Ziffern von z.B, 211-**162007** in HEXzahlen wandeln und eintragen? Nächste Frage: Einzeln? 1^01 oder im Doppelpack 16^
10? Hoffe das ist des Pudels Kern.

**Reply by OpenShorty** [reply]
Nein das musst du nicht. Der Hostname ist nur der Name mit dem sich deine DTU in deinem Netzwerk meldet so das du sie identifizieren kannst. Mein ausgegrauter Bereich ist der personalisierte Name meiner DTU. Bei Marc ist es der vorausgefüllte Name. Wie schon erwähnt, schicke uns einen Screenshot deiner Netzwerkeinstellung in der DTU sonst können wir dir wenig helfen. So gern wir möchten.

Und wenn ich mich recht entsinne, die DTU sucht und zeigt dir keine WLan Netzwerke an wie ein Handy. Du musst alles aktiv manuell eintragen damit ein Verbindungsversuch startet.

Und das was du einträgst muss exakt mit deiner FRITZ!Box übereinstimmen. Also SSID und Passwort

**Reply by Weiberhaus** [reply]
Moin Jungs, 
die mahnenden Worte von Shorty haben dazu geführt, das ich die SSID mit dem Zusatz AF ergänzt habe. Vorher FRITZ!Box 7530 jetzt 
2 Versuche mit FRITZBox! 7530 AF  und alternativ FRITZBox!7530AF...........leider nicht von Erfolg gekrönt. Irgendwo in den Dateien habe ich gesehen, das ich schon an die 20 Konfigurationen geschafft habe. Ich kenne mich mit der Speicherfiguration des ESP nicht aus. Nicht das da schon die Kapazitäten ausgelutscht sind. Zurück auf Anfang? Werkseinstellungen? IM Anhang die Screenshots. Wünsche schönen Tag.
![IMG-20240314-WA0003](https://github.com/tbnobody/OpenDTU/assets/162921779/d5b926f0-520e-443f-8df2-5686e9e84913)
![IMG-20240314-WA0002](https://github.com/tbnobody/OpenDTU/assets/162921779/3ddf6a13-ec0f-418b-be99-3318503d

**Reply by OpenShorty** [reply]
Moin, da kommen wir der Sache doch schon näher. Sag mal der Name FRITZBox!7530AF ist auch wirklich der Name wie er in der Fritzbox als SSID eingetragen ist? Aber iwie hört sich da alles komisch an. Ist dein Smartphone auch mit dem gleichen WLan verbunden?
Wenn du auf deinem Smartphone dein WLAN Netzwerk suchst, wie heisst es denn da? 
Alternativ kannst du auch in deiner Fritzbox schauen unter WLAN>Funknetz>Name des Wlan Funknetzes? Was da drin steht muss exakt auch in der DTU eingetragen werden. Ich haben das Gefühl das hier der Fehler vergraben ist. 

Wenn du schon dabei bist, schaue doch auch mal unter WLAN>Verschlüsselung>WLAN Zugang beschränken ob da die Option "Alle neuen WLAN-Geräte zulassen aktiviert ist und der Punkt darunter deaktiviert ist.

**Reply by Weiberhaus** [reply]
Moin!
Mich laust der Affe: FRITZBox7530  .......... ohne Ausrufungszeichen. Das Allerschönste ist: Ich komme mit meinem Wlan Passwort
(Original, hinter der Box aufgeklebt) nicht mehr ins Netz. Falsches Passwort! irgendwie liegt hier der Hase im Korn.

**Reply by Weiberhaus** [reply]
Manchmal sieht man den Wald vor lauter Bäumen nicht. Ich bin Drinnen. Habe das Fritzbox KENNWORT 100 Mal anstelle des Netzwerkschlüssels eingegeben. Danke für eure Unterstützung. Habe mir übrigens eine Arduino Grundausstattung zugelegt...........
wenn ich nicht weiterkomme, weiß ich nun, wo ich mich melden kann.

**Reply by OpenShorty** [reply]
Na sehr schön das es geklappt hat und wir dir helfen konnten.
